tools/gulp/gulpfile.ts

```diff
@@ -1,16 +1,21 @@
+import {createPackageBuildTasks} from './util/package-tasks';
+
+/** Create gulp tasks to build the different packages in the project. */
+createPackageBuildTasks('cdk');
+createPackageBuildTasks('material', ['cdk']);
+createPackageBuildTasks('material-examples', ['material']);
```

Comment:
The project/repo is already _called_ "material", though; I'd say that part of the name is implicit. As someone working on it, my mental model is just "I want to build the examples" -> "build:examples".